webrtc.html

```diff
@@ -1316,6 +1393,15 @@
                     method was invoked.</p>
                   </li>
```

Comment:
Good catch. Sidenote: it may be useful to define a **check that the connection is open** shorthand, since the [[isClosed]] check is used so often.